setup.sh

```diff
@@ -130,36 +136,45 @@ elif [[ $MODE == "nightly" ]]; then
     pip3 install tbp-nightly --upgrade
 elif [[ $MODE == "head" ]]; then 
 # Head mode
+    echo "Installing jax and jaxlib from head"
```

Comment:
Why are you using this mode? Elsewhere we use nightly. (Tony and I were talking about getting rid of this.)